### ISSUE TITLE
Tree view breaks line on paste action

### ISSUE THREAD

**Issue by Glumanda99** [contao/core-bundle#1143]
If you use an extension which will add actions to the listing such as [hofff/contao-opengraph](https://github.com/hofff/contao-opengraph) does, the whole `.tl_right` container will break to the next line. I played around with the backend css and removed the `width: calc(100% - 240px)` from `.tl_left` and it seems to fix this issue. I don't know why you even set a width there at all.

![gg37izx](https://user-images.githubusercontent.com/1057526/31575905-d2d7a82c-b0f0-11e7-977c-0b59e90e0767.png)

**Comment by leofeyer** [comment]
Because without the `width`, it looks like this:

<img width="1073" alt="" src="https://user-images.githubusercontent.com/1192057/32602096-271a346e-c545-11e7-89ea-1cd261554b4b.png">

**Comment by leofeyer** [comment]
We could decrease the width of `tl_left` so instead of this

<img width="1071" alt="" src="https://user-images.githubusercontent.com/1192057/32602163-6f13b16e-c545-11e7-8faa-da78833419bc.png">

it looks like this

<img width="1076" alt="" src="https://user-images.githubusercontent.com/1192057/32602187-81ebedf6-c545-11e7-9e76-2555c8f32f13.png">

However, the more icons are added in `tl_right` the higher is the chance of the problem reoccurring. @contao/developers Any ideas?

**Comment by fritzmg** [comment]
Imho not defining a width looks fine to me.

**Comment by Glumanda99** [comment]
@leofeyer I see your point here, but I don't know if that long site names make sense?

How about to decrease the width and [truncate lines](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/):

![screenshot_2017-11-09_12-53-53](https://user-images.githubusercontent.com/1057526/32604259-51df6f7c-c54d-11e7-9e98-5fc36b78a113.png)

**Comment by fritzmg** [comment]
@Glumanda99 this would still limit the amount of buttons you can have before another line break occurs

**Comment by fritzmg** [comment]
This would not be a problem, if `.tl_listing .tl_left` did not have `float:left;` and `.tl_right` came before `.tl_left` in the markup. Then `.tl_right` can float right with the text automatically breaking before the icons.

**Comment by leofeyer** [comment]
This is not much better, only the text is now above the buttons instead of below.

A better solution would be removing the floats and using flexbox instead:

```css
li {
    display:flex;
}
li .tl_left {
    flex-grow:1;
}
li.tl_right {
    padding-left:9px;
    white-space:nowrap;
}
```

<img width="1070" alt="" src="https://user-images.githubusercontent.com/1192057/32608623-374db5c8-c55d-11e7-8fa7-490802f865b7.png">

**Comment by leofeyer** [comment]
Fixed in 94cb211b39b21fc60164219d62491747b88221d8.
